.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,25 +6,31 @@ labels: 'bug: pending triage'
 assignees: ''
 ---
 
-<!--
-中文用户请注意：请仔细阅读以下模版，如果不遵循模版，issue 将会被直接关闭。
--->
+<!-- 中文用户请注意：请仔细阅读以下模版，如果不遵循模版，issue 将会被直接关闭。 -->
+<!-- IMPORTANT: Please do not ignore this template. If you do, your issue will be closed immediately. -->
 
-**⚠️ IMPORTANT ⚠️ Please do not ignore this template. If you do, your issue will be closed immediately.**
+### Before submitting the issue, please make sure you do the following
 
-- [ ] Read [the docs](https://vitejs.dev/guide/).
+- [ ] Read the [Contributing Guidelines](https://github.com/vitejs/vite/blob/main/.github/contributing.md).
+- [ ] Read the [docs](https://vitejs.dev/guide).
+- [ ] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
+- [ ] Provide a description in this issue that describes the bug.
 - [ ] Make sure this is a Vite issue and not a framework-specific issue. For example, if it's a Vue SFC related bug, it should likely be reported to https://github.com/vuejs/vue-next instead.
 - [ ] This is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitejs/vite/discussions) or join our [Discord Chat Server](https://chat.vitejs.dev/).
 
+---
+
 ## Describe the bug
 
-A clear and concise description of what the bug is.
+<!-- A clear and concise description of what the bug is. -->
 
 ## Reproduction
 
-Please provide a link to a repo that can reproduce the problem you ran into.
+<!--
+  Please provide a link to a repo that can reproduce the problem you ran into.
 
-A reproduction is **required** unless you are absolutely sure that the the problem is obvious and the information you provided is enough for us to understand what the problem is. If a report has only vague description (e.g. just a generic error message) and has no reproduction, it will receive "need reproduction" label. If no reproduction is provided after 3 days, it will be auto-closed.
+  A reproduction is required unless you are absolutely sure that the problem is obvious and the information you provided is enough for us to understand what the problem is. If a report has only vague description (e.g. just a generic error message) and has no reproduction, it will receive "need reproduction" label. If no reproduction is provided after 3 days, it will be auto-closed.
```

Comment:
**thought:** This text is really long, a user could ignore it cause of that
Can someone help me to trim this down to a maximum of 2 lines of text?

Reply:
I would leave the bolding in required. Reduced a bit the wording, I don't think much can be done

> A reproduction is **required** unless you are absolutely sure that the issue is obvious and the provided information is enough to understand the problem. If a report is vague (e.g. just a generic error message) and has no reproduction, it will receive a "need reproduction" label. If no reproduction is provided after 3 days, it will be auto-closed.

Reply:
The bolding wouldn't be visible or rendered as bold, cause it is in a comment